source/buildspec.yml

```diff
@@ -0,0 +1,39 @@
+version: 0.2
+
+env:
+    variables:
+      ARTIFACT_BUCKET: "login-gov.lambda-functions.894947205914-us-west-2"
```

Comment:
is this a real bucket ID or an example? if it's real I think we need to remove it

I can go put it in CircleCI secrets/config if you don't have access

Reply:
It is unused and has been removed.